pkg/controller/utils/datadog/datadog.go

```diff
@@ -0,0 +1,451 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+
+package datadog
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"hash/fnv"
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	datadoghqv1alpha1 "github.com/DataDog/datadog-operator/pkg/apis/datadoghq/v1alpha1"
+	"github.com/DataDog/datadog-operator/pkg/controller/utils"
+	api "github.com/zorkian/go-datadog-api"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+const (
+	defaultSendMetricsInterval  = 15 * time.Second
+	defaultMetricsNamespace     = "datadog.operator"
+	gaugeType                   = "gauge"
+	deploymentSuccessValue      = 1.0
+	deploymentFailureValue      = 0.0
+	deploymentMetricFormat      = "%s.%s.deploymentsuccess"
```

Comment:
```suggestion
	deploymentMetricFormat      = "%s.%s.deployment.success"
```
what do you think about this suggestion?

Reply:
agreed